tests/loaders/schema/schema-from-typedefs.spec.ts

```diff
@@ -51,6 +51,20 @@ describe('schema from typedefs', () => {
     expect(schema.getTypeMap()['Query']).toBeDefined();
   });
 
+  it('should work without globs correctly', () => {
```

Comment:
Hello @darkbasic ! Could you also add the same test but in `integration.spec.ts`? So we know it won't break the chain of loaders and load schema correctly?

Reply:
Done